src/app/chat/core.nim

```diff
@@ -43,6 +43,10 @@ proc handleChatEvents(self: ChatController) =
     self.view.updateChats(evArgs.chats)
     self.view.pushMessages(evArgs.messages)
 
+  self.status.events.on("chatHistoryCleared") do(e: Args):
+    var args = ChannelArgs(e)
+    self.view.clearMessages(args.chat.id)
+
```

Comment:
First wanted to go with the `chatUpdated` event but that one only allows for adding new channels and messages, so I decided to introduce a new event.

Lemme know if this needs change